lib/number.js

```diff
@@ -8,3 +8,25 @@ var utils = module.exports;
 utils.intToHex = function(num) {
   return '#' + (num).toString(16);
 };
+
+/**
+ * Gives a number between a min & a max (inclusive) with an optional precision
+ * @param  {Number} min       Minimum num
+ * @param  {Number} max       Maximum num
+ * @param  {Number} precision Result precision
+ * @return {Number}           Random number between the two (inclusive)
+ */
+utils.between = function(min, max, precision) {
+  precision = precision || 0;
+
+  var multiplier = Math.pow(10, precision);
+  var swap;
+
+  if (min > max) {
+    swap = min;
+    min = max;
+    max = swap;
+  }
+
+  return Math.round((Math.random() * (max * multiplier - min * multiplier)) + (min * multiplier)) / multiplier;
+};
```

Comment:
`.between` made me think it was the median number between 2 numbers. Maybe `.random` ?